TodoList/View Controllers/TodoListViewController.swift

```diff
@@ -27,10 +33,7 @@ class TodoListViewController: UITableViewController {
 
         tableView.register(cellType: ItemTableViewCell.self)
         tableView.tableFooterView = footer
-        tableView.estimatedRowHeight = 44
-        tableView.rowHeight = UITableViewAutomaticDimension
-        tableView.estimatedSectionFooterHeight = 80
-        tableView.sectionFooterHeight = UITableViewAutomaticDimension
+        tableView.keyboardDismissMode = .interactive
```

Comment:
That's so cool. 
Have been using `UIScrollViewDelegate` to dismiss keyboard.

Reply:
Yeah, I love finding these tricks. I try to find the simplest way that uses the least code possible.